deltachat-ffi/Doxyfile.css

```
@@ -1,14 +1,24 @@
 
+:root {
+	--accent: hsl(0 0% 70%);
```

this is a smart way of doing things, nice!

however, the colors are a bit too dark/light, i tweaked them a bit and moved them closer to what similar dark themes are doing:

```suggestion
	--accent: hsl(0 0% 85%);
```